tests/Feature/Commands/MakeMailForceTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Tests\Feature\Commands;
+
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\File;
+
+it(
+    description: 'can force generate exists mail class',
+    closure: function () {
+        Artisan::call('diamond:install');
+        Artisan::call('diamond:mail UserApproved User');
+        Artisan::call('diamond:mail UserApproved User --force');
```

Comment:
before you run this artisan make sure if didn't using --force cannot replace the fail like make an assert for non --force

1. Run diamond:mail UserApproved User
2. Run again diamond:mail UserApproved User > assert false
3 diamond:mail UserApproved User --force > assert true